tutorials/cp-trial-quick-onboarding/cp-trial-quick-onboarding.md

```diff
@@ -132,60 +152,8 @@ To integrate services with applications, the service credentials must be deliver
     ![Cloud Foundry service instances table](cf-bind-instance.png)
 
 [VALIDATE_4]
-
-[ACCORDION-END]
-
-[ACCORDION-BEGIN [Step 5: ](Navigation in the cockpit)]
-
-There are 2 ways of navigating in the cockpit:
-
-- using the left hand-side navigation
-- using the breadcrumbs at the top of the page
-
-Remember all that stuff about global accounts, subaccounts we talked about earlier? There are certain actions you can perform at each level - we will refer to the sum of these actions as a **scope**. For example, when you are in a global account you can add global account members, create subaccounts and add entitlements to subaccounts. They represent the **global account scope**.
-
-This scope is reflected in both the left hand-side navigation and the breadcrumbs. This step helps you understand how to use both of them together to navigate through the cockpit in the most efficient way possible.
-
-### Left hand-side navigation
-
- The left hand-side navigation changes depending on where you are in the cockpit. It reflects the actions possible in that scope. You can see some examples below:
-
-**Cloud Foundry Subaccount Scope:**
-
-![Cloud Foundry subaccount scope](cf-sa-scope.png)
-
-**Cloud Foundry Space Scope:**
-
-![Cloud Foundry space scope](cf-space-scope.png)
-
-
-### Breadcrumbs
-
-You may ask yourself - if the left hand-side navigation changes depending on where you are in the cockpit,  then how can you navigate back? Simple - you use the breadcrumbs at the top.
-
-The breadcrumbs coincide with the hierarchical path you've travelled in order to reach your current scope in the cockpit. For example, to reach an application in the Cloud Foundry environment you will have to go to your global account, then to the subaccount, then to the space and then finally to the application.
-
-The last item in your breadcrumbs always represents the scope you find yourself in at that moment. To go back, simply follow the breadcrumbs to retrace your steps.
-
-Let's have a look at the following screenshot, taken from an application deployed in the Cloud Foundry dev space:
-
-![Breadcrumbs](cf-breadcrumbs.png)
-
-1. **Trial Home** - here you can find your trial homepage
-
-2. **Global account** - this contains all your subaccounts and entitlements
-
-3. The 3rd item represents your **subaccount**.
-
-4. Afterwards, you have your **space** as the 4th item.
-
-5. Finally, you have your **deployed application** - this is the current scope.
-
-[VALIDATE_5]
-
 [ACCORDION-END]
-
-[ACCORDION-BEGIN [Step 6: ](Guided tours in the cockpit)]
+[ACCORDION-BEGIN [Step : ](Guided tours in the cockpit)]
```

Comment:
I would remove this step completely since the guided tours are now a lot more prominent than before.